fnal_column_analysis_tools/processor/dataframe.py

```diff
@@ -13,7 +13,7 @@ class LazyDataFrame(MutableMapping):
     """
     def __init__(self, tree, stride=None, index=None, preload_items=None):
         self._tree = tree
-        self._branchargs = {'awkwardlib': awkward}
+        self._branchargs = {'awkwardlib': awkward, 'flatten': True}
```

Comment:
@nsmith- FYI this is needed to make it so that the same analysis code works everywhere. I have to manually flatten jagged arrays in the spark version, and it's less confusing when you don't have to do .content on everything going into a JaggedCandidateArray.

Also makes it seemless with striped (in the future).

Reply:
Hm ok, you'll have to update `binder/muonspectrum_v3.ipynb` as well, then.

Reply:
Shall do, we can revisit this choice when we free ourselves of pandas.

Reply:
This does completely disable usage of array branches where no `nObject` branch exists.  Could it be a configurable thing?  Recall that in striped, there was a magic member in each `ArrayObject` for the counts array.

Reply:
Making it configurable is no problem.
It'll also not really be needed when we no longer need to go through pandas and can just awkward.fromarrow() things.

Reply:
But I think this does force us to conform to a certain standard when we want something to scale out.... I think it's OK to eat that for now?